From 6fd50edca6372aa255c96207ea18c52a85a4724f Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Wed, 27 Jan 2016 19:49:58 +0800
Subject: [PATCH 5/6] Add Mediatek proprietary format

Add DRM_FORMAT_MT21 for Mediatek Properity video block format.

Signed-off-by: Chiawen Lee <chiawen.lee@mediatek.com>
---
 include/drm/drm_fourcc.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/drm/drm_fourcc.h b/include/drm/drm_fourcc.h
index 63a80ca..8061c01 100644
--- a/include/drm/drm_fourcc.h
+++ b/include/drm/drm_fourcc.h
@@ -107,6 +107,8 @@
 #define DRM_FORMAT_NV16		fourcc_code('N', 'V', '1', '6') /* 2x1 subsampled Cr:Cb plane */
 #define DRM_FORMAT_NV61		fourcc_code('N', 'V', '6', '1') /* 2x1 subsampled Cb:Cr plane */
 
+#define DRM_FORMAT_MT21		fourcc_code('M', 'T', '2', '1') /* Mediatek Block Mode */
+
 /*
  * 3 plane YCbCr
  * index 0: Y plane, [7:0] Y
-- 
2.7.0.rc3.207.g0ac5344

